### PR TITLE
chore: adding mainnet chain id and setting as default

### PR DESCRIPTION
**Pull request by distractedm1nd**
:)
Also removed blockspace race
Part of #2823

### CODE REVIEW

**Review by ramin — APPROVED** [review]

File: nodebuilder/p2p/network.go [hunk 1]
```diff
@@ -52,20 +52,20 @@ func (n Network) String() string {
 
 // networksList is a strict list of all known long-standing networks.
 var networksList = map[Network]struct{}{
-	Arabica:        {},
-	Mocha:          {},
-	BlockspaceRace: {},
-	Private:        {},
+	Mainnet: {},
```
Comment (Wondertan) [inline nodebuilder/p2p/network.go:55]: [discussion]
We may wanna call it just Celestia. We don't have `Testnet` and we have names for all other testnets.

Comment (ramin) [reply nodebuilder/p2p/network.go:55]: hmmmm, yes i think i agree with this

Comment (renaynay) [reply nodebuilder/p2p/network.go:55]: p2p.Celestia

I think I _prefer_ `mainnet` but I'm fine with whatever. Not a big deal to me

**Review by renaynay — APPROVED** [review]

File: nodebuilder/p2p/network.go (hunk 1 above)
Comment (distractedm1nd) [reply nodebuilder/p2p/network.go:55]: I like Mainnet more but am also not very opinionated. in the worst case we can change it later if we don't like it without breaking anything

Comment (ramin) [reply nodebuilder/p2p/network.go:55]: agree, don't feel very strong either way. BUT, with naming i always err on the side of "what do we say when we talk about it" and we're much more likely to say "on Arabica" or "if we are on Mainnet then we should remove the suffix" vs "if we are on Celestia". So....maybe just rolling with Mainnet is correct

Comment (distractedm1nd) [reply nodebuilder/p2p/network.go:55]: also as rene said, I feel better about p2p.Mainnet naming than p2p.Celestia, which might be confusing

Comment (Wondertan) [reply nodebuilder/p2p/network.go:55]: To me, it's similarly confusing as p2p.Arabica

Comment (Wondertan) [reply nodebuilder/p2p/network.go:55]: > "if we are on Celestia"

When we say on Celestia it should mean on mainnet by default IMO

**Pull request merged**